opt/Makefile

```diff
@@ -77,6 +77,9 @@ device selection options (fetch, build, and test):
   GPU=1         # build for GPU
   CUDA=1        # build for GPU
   JETSON=1      # build for Nvidia Jetson
+
+enterprise variant
+  LITE=1        # build RedisAI LITE
```

Comment:
Can we maintain the use of the REDISAI_ prefix for LITE? This would be some way to differentiate that it's at least an internal need.